core/src/test/scala/unit/kafka/server/DynamicBrokerConfigTest.scala

```diff
@@ -965,6 +965,31 @@ class DynamicBrokerConfigTest {
     verifyNoMoreInteractions(remoteLogManager)
   }
 
+  @Test
+  def testEnableFollowerFetchLastTieredOffset(): Unit = {
+    val props = TestUtils.createBrokerConfig(0, TestUtils.MockZkConnect, port = 9092)
+    val config = KafkaConfig.fromProps(props)
+    val serverMock: KafkaServer = mock(classOf[KafkaServer])
```

Comment:
can we mock the trait KafkaBroker instead?

```suggestion
    val serverMock: KafkaBroker = mock(classOf[KafkaBroker])
```

Reply:
Done